atomic_reactor/plugins/exit_koji_promote.py

```diff
@@ -553,12 +554,19 @@ def get_metadata(self):
         :return: tuple, the metadata and the list of Output instances
         """
         try:
-            metadata = get_build_json()["metadata"]
+            build_json = get_build_json()
```

Comment:
I think it'd be cleaner to retrieve the `build_json` outside of the try-block since it's used later outside of it.